src/stream/src/from_proto/hash_join.rs

```diff
@@ -144,6 +155,8 @@ impl ExecutorBuilder for HashJoinExecutorBuilder {
             degree_state_table_l,
             state_table_r,
             degree_state_table_r,
+            left_deduped_input_pk_indices,
+            right_deduped_input_pk_indices,
```

Comment:
Those are already in `params_r` /  `params_l`. (I somehow forgot to use it..)

Maybe also delete the `deduped_pk_indices` in `JoinSide` since it is not in use now.

Reply:
> Those are already in `params_r` / `params_l`. (I somehow forgot to use it..)
> 
> Maybe also delete the `deduped_pk_indices` in `JoinSide` since it is not in use now.

fixed